spec/controllers/chapters_controller_spec.rb

```diff
@@ -4,74 +4,554 @@
+    end
+
+    it "does not assign @next_chapter when on last chapter" do
+      chapter = create(:chapter, work: work, authors: work.authors, position: 2, posted: true)
+      get :show, work_id: work.id, id: chapter.id
+      expect(assigns[:next_chapter]).to be_nil
+    end
+
+    it "assigns @comments to only reviewed comments" do
+      moderated_work = create(:work, posted: true, moderated_commenting_enabled: true)
+      comment = create(:comment, commentable_type: "Chapter", commentable_id: moderated_work.chapters.first.id)
+      comment.unreviewed = false
+      comment.save
+      create(:comment, unreviewed: true, commentable_type: "Chapter", commentable_id: moderated_work.chapters.first.id)
+      get :show, work_id: moderated_work.id, id: moderated_work.chapters.first.id
+      expect(assigns[:comments]).to eq [comment]
+    end
+
+    it "assigns @page_title with fandom, author name, work title, and chapter" do
+      expect_any_instance_of(ChaptersController).to receive(:get_page_title).with("Testing", user.pseuds.first.name, "My title is long enough - Chapter 1").and_return("page title")
+      get :show, work_id: work.id, id: work.chapters.first.id
+      expect(assigns[:page_title]).to eq("page title")
+    end
+
+    it "assigns @page_title with unrevealed work" do
+      allow_any_instance_of(Work).to receive(:unrevealed?).and_return(true)
+      get :show, work_id: work.id, id: work.chapters.first.id
+      expect(assigns[:page_title]).to eq("Mystery Work - Chapter 1")
+    end
+
+    it "assigns @page_title with anonymous work" do
+      allow_any_instance_of(Work).to receive(:anonymous?).and_return(true)
+      expect_any_instance_of(ChaptersController).to receive(:get_page_title).with("Testing", "Anonymous", "My title is long enough - Chapter 1").and_return("page title")
+      get :show, work_id: work.id, id: work.chapters.first.id
+      expect(assigns[:page_title]).to eq("page title")
+    end
+
+    it "assigns @kudos to non-anonymous kudos" do
+      kudo = create(:kudo, commentable_id: work.id, pseud: create(:pseud))
+      create(:kudo, commentable: work)
+      get :show, work_id: work.id, id: work.chapters.first.id
+      expect(assigns[:kudos]).to eq [kudo]
+    end
+
+    it "assigns instance variables correctly" do
+      second_chapter = create(:chapter, work: work, authors: work.authors, position: 2, posted: true)
+      third_chapter = create(:chapter, work: work, authors: work.authors, position: 3, posted: true)
+      comment = create(:comment, commentable_type: "Chapter", commentable_id: second_chapter.id)
+      kudo = create(:kudo, commentable_id: work.id, pseud: create(:pseud))
+      tag = create(:fandom)
+      expect_any_instance_of(Work).to receive(:tag_groups).and_return("Fandom" => [tag])
+      expect_any_instance_of(ChaptersController).to receive(:get_page_title).with("The 1 Fandom", user.pseuds.first.name, "My title is long enough - Chapter 2").and_return("page title")
```

Comment:
Oh, I think I see why this might be failing -- this expects "The 1 Fandom," but other specs that run before this probably created fandom tags, which means the number is off:
```
expected: ("The 1 Fandom", "obonss2d142", "My title is long enough - Chapter 2")
              got: ("The 4 Fandom", "obonss2d142", "My title is long enough - Chapter 2")
```
 You might have to do something like `tag.name` here instead.